tests/PKSim.Tests/Presentation/IndividualSimulationParametersPresenterSpecs.cs

```diff
@@ -15,15 +16,17 @@ public abstract class concern_for_IndividualSimulationParametersPresenter : Cont
    {
       protected IParameterGroupsPresenter _parameterGroupPresenter;
       protected ISimulationParametersView _view;
+      protected IInteractiveSimulationRunner _interactiveSimulationRunner;
       protected IndividualSimulation _simulation;
 
       protected override void Context()
       {
          _view = A.Fake<ISimulationParametersView>();
          _parameterGroupPresenter = A.Fake<IParameterGroupsPresenter>();
+         _interactiveSimulationRunner = A.Fake<IInteractiveSimulationRunner>();
          _simulation = new IndividualSimulation {Model = A.Fake<IModel>()};
          A.CallTo(() => _parameterGroupPresenter.View).Returns(A.Fake<IParameterGroupsView>());
-         sut = new IndividualSimulationParametersPresenter(_view, _parameterGroupPresenter);
+         sut = new IndividualSimulationParametersPresenter(_view, _parameterGroupPresenter, _interactiveSimulationRunner);
```

Comment:
should we test the behavior at least once ?E.g. handle of events works as exepcted?